src/clarity-angular/datagrid/_datagrid.clarity.scss

```diff
@@ -266,6 +266,28 @@
             }
         }
 
+        .datagrid-footer-select.checkbox {
+            line-height: inherit;
+            margin-right: auto;
+
+            label {
+                color: unset;
+                cursor: default;
+                opacity: 1;
+            }
+
+            input[type="checkbox"] + label {
+                &::before, &::after {
+                    top: 8px;
```

Comment:
I need to refactor this in checkbox.clarity.scss. I should have probably added a TODO but for now the hardcoded pixels is fine.